lib/smart_app_launch/client_suite/token_request_bsca_verification_test.rb

```diff
@@ -17,7 +17,16 @@ class SMARTClientTokenRequestBackendServicesConfidentialAsymmetricVerification <
     title 'Verify SMART Token Requests'
     description %(
         Check that SMART token requests are conformant.
-      )
+    )
+    verifies_requirements 'hl7.fhir.uv.smart-app-launch_2.2.0@229',
+                          'hl7.fhir.uv.smart-app-launch_2.2.0@230',
+                          'hl7.fhir.uv.smart-app-launch_2.2.0@233',
+                          'hl7.fhir.uv.smart-app-launch_2.2.0@234',
```

Comment:
I don't see @234 tested here. Please confirm and remove if you agree.

@234
After generating this authentication JWT, the client requests an access token via HTTP `POST` to the FHIR authorization server’s token endpoint URL, using content-type `application/x-www-form-urlencoded`

Reply:
Removed in [9b4d4ad](https://github.com/inferno-framework/smart-app-launch-test-kit/pull/105/commits/9b4d4adf0a9945b994d01c6ee21808ba05be91a6).